lib/matplotlib/figure.py

```diff
@@ -1,4 +1,4 @@
-"""
+:doc:"""
```

Comment:
too much markup :p

Reply:
👍 Yeah, that didn't work out too well ;-)